masterlist.yaml

```diff
@@ -455,8 +461,9 @@ plugins:
       - crc: 0x6F103898
         util: SSEEdit v3.2.1
   - name: 'BS_DLC_patch.esp'
-    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/10917/']
-    # Beyond Skyrim - DLC Integration Patch
```

Comment:
Would it be better to leave these comments just for patches?

Reply:
I think the comment is useful because I wouldn't know what BS stood for.

Reply:
Okay will leave them in where appropriate.